editors/vscode/slint.tmLanguage.json

```diff
@@ -13,31 +13,31 @@
                     "include": "#general"
                 },
                 {
-                    "match": "\\b(import|from|export|struct|enum|global|component|inherits)\\b",
+                    "match": "\\b[^-](import|from|export|struct|enum|global|component|inherits)\\b[^-]",
```

Comment:
This will match too much if i'm not mistaken

for example `import{foo}from"bar";`
and it will recognize `import{`  (or nothing at all because there is not something, and `from"`
Oh, and maybe  `property <bool> fimport`  and it will recognize fimport

Reply:
I asked a AI model for a regexp that would fix the problem and it gave me this:
`"(?<![\\w-])(import|from|export|struct|enum|global|component|inherits)(?![\\w-])"`  but I haven't tried.

Reply:
but [^-] matches the character and include it in the match.
so it should match `fimport`  because [^-] matches f and the rest is matched by the keyword